scripting/z_esc_reference.rst

```diff
@@ -183,8 +184,8 @@ Groups cann also use conditions:
 Blocking
 ~~~~~~~~
 
-Some commands will block execution of the event until they finish,
-others won’t. See the command’s reference for details on which commands
+Some commands will block execution of the event until they finish; 
```

Comment:
Hm. Is this semicolon correct there? Wouldn't a comma be the thing to use here?

Reply:
It could probably also work, but a semi-colon here reinforces a break in the sentence and makes the dichotomy of what you're trying to express very clear. I'd prefer a semi-colon here to a comma, but will defer to you, of course.

Reply:
Okay, let's keep your change.